funcs/historical.js

```diff
@@ -36,6 +35,7 @@ var historical = async (keys, redis) => {
   // to store parsed data
   const result = [];
   const timelineKey = parsedCases[0].splice(4);
+  // REVIEW
   // parsedCases.pop()
   // parsedDeaths.pop()
   // recParsed.pop()
```

Comment:
```suggestion
```
We should either remove these lines or add some meaningful comment about why these lines should remain commented out.